src/variants/with-taste-skill/gpt-6-astra/source/app/globals.css

```
@@ -0,0 +1,474 @@
+@media (max-width: 767px) {
+  .page-width { width: calc(100% - 40px); }
+  .site-header { height: 72px; gap: 15px; }
+  .brand { font-size: 26px; }
+  .brand > svg { width: 26px; }
+  .desktop-nav { display: none; }
+  .nav-cta { font-size: 10px; min-height: 34px; padding: 8px 12px; }
+  .nav-cta svg { width: 15px; }
+  .header-actions { gap: 8px; }
+  .mobile-menu { display: block; position: relative; }
+  .mobile-menu > .icon-button { width: 32px; height: 32px; border: 0; background: transparent; }
+  .menu-lines { width: 19px; height: 9px; border-block: 1.5px solid currentColor; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)